index.html

```diff
@@ -18,21 +20,49 @@
     <div class="lnr lnr-menu"></div>
     <nav id="main-nav" class="nav-menu">
       <ul>
-        <li>menu 1 </li>
-        <li>menu 2 </li>
-        <li>menu 3 </li>
+        <a href="#"><li class="tab"><i class="fa fa-home fa-fw"></i> Home </li></a>
+        <a href="#"><li class="tab"  data-content="about-me"><i class="fa fa-user fa-fw"></i> About Me</li></a>
+        <a href="#"><li class="tab" data-content="projects"><i class="fa fa-product-hunt fa-fw"></i> Projects</li></a>
       </ul>
     </nav>
   </section>
 </header>
     <main>
-      <p>main Body will go here</p>
+      <section id="about-me" class="data-content" class="tab-content">
+        <h1>About me</h1>
+        <p>Some information about me will go here</p>
+      </section>
+      <section id="projects" class="data-content" class="tab-content">
```

Comment:
it's a little confusing that the class on this section is the same as the data-content attribute on the tabs. I would rename or just take out since you're currently not using this class in your css.